core/mm/fobj.c

```diff
@@ -759,7 +762,7 @@ struct fobj *fobj_sec_mem_alloc(unsigned int num_pages)
 	if (MUL_OVERFLOW(num_pages, SMALL_PAGE_SIZE, &size))
 		goto err;
 
-	f->mm = tee_mm_alloc(&tee_mm_sec_ddr, size);
+	f->mm = phys_mem_ta_alloc(size);
```

Comment:
This is embedded only when pager is enabled, as for other changes in this source file. Wouldn't it be more consistent is `nex_phys_mem_ta_alloc()` is used here?

Reply:
Yes and no. The pager isn't supported with virtualization so it doesn't change anything, but we have quite a few places around here where non-nex functions are used. Either we'd have to fix all those too or it would only make things even more confusing. I'd also like to keep the scope to what this commit is about, getting rid of "tee_mm_sec_ddr".

I wouldn't mind a cleanup where the pager uses nex resources as needed, but I don't think that will be enough to get the pager to work with virtualization.